
From: Marcin Juszkiewicz <openembedded@haerwu.biz>

Card reported by Ångström user:
http://bugs.openembedded.net/show_bug.cgi?id=3236

Socket 1:
   product info: "Wireless LAN", "11Mbps PC Card", "Version 01.02", ""
   manfid: 0x0156, 0x0002
   function: 6 (network)

Signed-off-by: Marcin Juszkiewicz <openembedded@haerwu.biz>
Acked-by: Pavel Roskin <proski@gnu.org>

---
 drivers/net/wireless/hostap/hostap_cs.c |    3 +++
 1 file changed, 3 insertions(+)

--- linux-2.6.24.orig/drivers/net/wireless/hostap/hostap_cs.c
+++ linux-2.6.24/drivers/net/wireless/hostap/hostap_cs.c
@@ -892,10 +892,13 @@ static struct pcmcia_device_id hostap_cs
 		0xa21501a, 0x59868926, 0xc9049a39),
 	PCMCIA_DEVICE_PROD_ID1234(
 		"The Linksys Group, Inc.", "Wireless Network CF Card", "ISL37300P",
 		"RevA",
 		0xa5f472c2, 0x9c05598d, 0xc9049a39, 0x57a66194),
+	PCMCIA_DEVICE_PROD_ID123(
+		"Wireless LAN" , "11Mbps PC Card", "Version 01.02",
+		0x4b8870ff, 0x70e946d1, 0x4b74baa0),
 	PCMCIA_DEVICE_NULL
 };
 MODULE_DEVICE_TABLE(pcmcia, hostap_cs_ids);
 
 
